### PR TITLE
feat(pack): add prisma language pack

### PR DESCRIPTION
**Pull request by 20k-ultra**
This follows the official prisma [docs](https://www.prisma.io/docs/guides/development-environment/editor-setup#vim) which mention to use https://github.com/pantharshit00/coc-prisma which in turn says to also use https://github.com/prisma/vim-prisma.

### CODE REVIEW

**Review by taskylizard — APPROVED** [review]

**Review by mehalter — COMMENTED** [review]
Could you please format the code with `stylua` to pass the tests?

File: lua/astrocommunity/pack/prisma/prisma.lua [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      -- Ensure that opts.ensure_installed exists and is a table or string "all".
+      if not opts.ensure_installed then
+        opts.ensure_installed = {}
+      elseif opts.ensure_installed == "all" then
+        return
+      end
+      utils.list_insert_unique(opts.ensure_installed, { "prisma" })
```
Comment (mehalter) [inline lua/astrocommunity/pack/prisma/prisma.lua:12]:
```suggestion
      utils.list_insert_unique(opts.ensure_installed, "prisma")
```

**Review by mehalter — CHANGES_REQUESTED** [review]
Small code change but that and stylua and we can get this merged! Thanks for this contribution!

**Review by mehalter — APPROVED** [review]
LGTM

**Pull request merged**